edx_rest_framework_extensions/auth/jwt/tests/test_cookies.py

```diff
@@ -29,3 +30,19 @@ def test_get_setting_value(self, jwt_cookie_func, setting_name, setting_value):
     @ddt.unpack
     def test_get_default_value(self, jwt_cookie_func, expected_default_value):
         self.assertEqual(jwt_cookie_func(), expected_default_value)
+
+    def test_get_decoded_jwt_from_existing_cookie(self):
```

Comment:
hmm... It's unclear to me whether a white-box-test built upon mocks is worthwhile to test and maintain.  I generally prefer black-box-tests that test to the interface - rather than the innards of the implementation.

Reply:
I started with black-box, but liked the simplicity of the mocked one.  I put back the black-box test.